radixdlt-core/docker/scripts/fix-vulnerabilities.sh

```diff
@@ -22,6 +6,3 @@ apt remove -y libglib2.0-0 libglib2.0-bin libglib2.0-data
 
 #CVE-2013-7445 and others (neded to compile drivers)
 apt remove -y linux-libc-dev
-
-apt remove -y unzip wget
-apt autoremove -y
```

Comment:
We're no longer updating stuff (is that why we've moved to `RUN apt update -y` in the Docker file?); We're also no longer removing unzip, curl, wget etc. This potentially introduces attack surfaces.

Why was this change made? Is there something different about Ubuntu 22.04 which suggests we don't need this?

Reply:
1. I've moved `apt get update -y` from this `fix-vulnerabilities.sh` to `Dockerfile` just to improve maintainability. Previously we were relying on side-effects of this script in our Dockerfile (java sdk installation relied on this apt update). Same is true for `autoremove`.
2. I've removed `remove unzip wget` as it was only needed for this manual `curl` installation which is no longer needed as we can rely on `curl` shipped with Ubuntu 22.04 - it's no longer vulnerable to this CVE.